src/fabric/twist.rs

```diff
@@ -125,6 +115,36 @@ impl Fabric {
         faces
     }
 
+    pub fn join_faces(&mut self, alpha_id: UniqueId, omega_id: UniqueId) {
+        let (alpha, omega) = (self.face(alpha_id), self.face(omega_id));
+        let (mut alpha_ends, omega_ends) = (alpha.radial_joints(self), omega.radial_joints(self));
+        alpha_ends.reverse();
+        let (mut alpha_points, omega_points) = (
+            alpha_ends.map(|id| self.location(id)),
+            omega_ends.map(|id| self.location(id))
+        );
+        let links = [(0, 0), (0, 1), (1, 1), (1, 2), (2, 2), (2, 0)];
+        let (_, alpha_rotated) = (0..3)
+            .map(|rotation| {
+                let length: f32 = links
+                    .map(|(a, b)| alpha_points[a].distance(omega_points[b]))
+                    .iter()
```

Comment:
```suggestion
                    .iter()
                    .map(|(a, b)| alpha_points[a].distance(omega_points[b]))
```
Technically more efficient since you prevent creating a new array